schemes/musica/tuvx/musica_ccpp_tuvx_load_species.F90

```diff
@@ -0,0 +1,216 @@
+module musica_ccpp_tuvx_load_species
+  use ccpp_kinds,          only: kind_phys
+  use musica_ccpp_species, only: MUSICA_INT_UNASSIGNED
+
+  implicit none
+  private
+
+  public :: configure_tuvx_species, check_tuvx_species_initialization
+  
+  integer, protected, public :: index_cloud_liquid_water_content = MUSICA_INT_UNASSIGNED
+  integer, protected, public :: index_dry_air = MUSICA_INT_UNASSIGNED
+  integer, protected, public :: index_O2 = MUSICA_INT_UNASSIGNED
+  integer, protected, public :: index_O3 = MUSICA_INT_UNASSIGNED
+
+  ! Constants
+  ! Clould liquid water
+  character(len=*), parameter, public :: CLOUD_LIQUID_WATER_CONTENT_LABEL = &
+    'cloud_liquid_water_mixing_ratio_wrt_moist_air_and_condensed_water'
+  character(len=*), parameter, public :: CLOUD_LIQUID_WATER_CONTENT_LONG_NAME = &
+    'cloud water mass mixing ratio with respect to moist air plus all airborne condensates'
+  character(len=*), parameter, public :: CLOUD_LIQUID_WATER_CONTENT_UNITS = 'kg kg-1'
+  real(kind_phys),  parameter, public :: CLOUD_LIQUID_WATER_CONTENT_MOLAR_MASS = 0.018_kind_phys ! kg mol-1
+  ! Gas species - dry air, O2, O3
+  character(len=*), parameter, public :: DRY_AIR_LABEL = 'air'
+  character(len=*), parameter, public :: O2_LABEL = 'O2'
+  character(len=*), parameter, public :: O3_LABEL = 'O3'
+  character(len=*), parameter, public :: TUVX_GAS_SPECIES_UNITS = 'molecule cm-3'
+  real(kind_phys),  parameter, public :: SCALE_HEIGHT_DRY_AIR = 8.01_kind_phys ! km
+  real(kind_phys),  parameter, public :: SCALE_HEIGHT_O2 = 7.0_kind_phys       ! km
+  real(kind_phys),  parameter, public :: SCALE_HEIGHT_O3 = 7.0_kind_phys       ! km
+  !> Molar mass value of dry air is obtained from 'CAM-SIMA/src/utils/std_atm_profile.F90'
+  real(kind_phys),  parameter, public :: MOLAR_MASS_DRY_AIR = 0.0289644_kind_phys ! kg mol-1
+  real(kind_phys),  parameter, public :: MOLAR_MASS_O2 = 0.0319988_kind_phys      ! kg mol-1
+  real(kind_phys),  parameter, public :: MOLAR_MASS_O3 = 0.0479982_kind_phys      ! kg mol-1
```

Comment:
Hi @mattldawson, @nusbaume, after discussing with Kyle, I decided not to include the TUV-x gas species in the MICM configuration, as we are planning to move the parsing features to OpenAtmos, which will offer more flexibility. Currently, the MICM parser requires species and reaction JSON files, but TUV-x does not have reaction file.  I think it would be more efficient to use a separate configuration for TUV-x, as it requires additional component including cloud liquid water content and specific gas species such as O2, O3, and dry air. This module includes the constituents (or species) needed by TUV-x, with molar masses and scale heights hard-coded, as this file functions as a configuration. However, this setup may change in the future as the parser is moved to OpenAtmos.

Reply:
Thanks for letting me know @boulderdaze!  I think as long as MUSICA can tell CAM-SIMA what the composition of the atmosphere is (ignoring physics or dynamics-specific constituents) then whatever method(s) you all choose is fine with me!